package.json

```diff
@@ -43,7 +43,7 @@
     "@stream-io/flat-list-mvcp": "0.10.3",
     "@voximplant/react-native-foreground-service": "3.0.2",
     "base-64": "1.0.0",
-    "commonmark": "npm:@mattermost/commonmark@0.30.1-3",
+    "commonmark": "github:mattermost/commonmark.js#bdd88ab24a012d0cff75708d7b61fcfc499366f7",
```

Comment:
To update once released to npm

Reply:
Yep, that's the plan. We also have to merge that PR first